engine/src/main/java/org/terasology/logic/console/commands/CoreCommands.java

```diff
@@ -523,17 +538,16 @@ public String spawnPrefab(@Sender EntityRef sender, @CommandParam("prefabId") St
         } else {
             dir.set(Direction.FORWARD.getVector3f());
         }
-        Quat4f rotation = Quat4f.shortestArcQuat(Direction.FORWARD.getVector3f(), dir);
```

Comment:
The cleanup of the method is appreciated, but this line seems like it has just been removed. Is it actually not needed, because it looks like it's used in spawning the prefab.

Reply:
Fixed , Turns out I was using a slightly older branch which had an outdated version of this method , have replaced it with the correct implementation along with other changes